docs/htdocs/info/docs/tools/vep/online/input.html

```diff
@@ -623,6 +617,13 @@ <h2 id="prediction">Predictions</h2>
                   Equivalent to the VEP plugin <a rel="external" 
                   href="https://github.com/Ensembl/VEP_plugins/blob/release/[[SPECIESDEFS::ENSEMBL_VERSION]]/AncestralAllele.pm">AncestralAllele</a>.</p>
                 </li>
+
+
+                <li><b>Enformer</b>
+                  <p>Predictions of variant impact on gene expression.
+                  This functionality is provided by the  <a rel="external" 
+                  href="https://github.com/Ensembl/VEP_plugins/blob/release/[[SPECIESDEFS::ENSEMBL_VERSION]]/Enformer.pm">Enformer</a>plugin.</p>
+                </li>
             </ul>
         </div>
     </div>
```

Comment:
on line 634 at the end between `</fieldset>` and `</div></div>`
```
..type="checkbox"></p></div></div></fieldset></div></div>
```

you can add the following HTML -
```
<fieldset><h2>Regulatory impact</h2><div class="_stt_Homo_sapiens form-field"><label for="CJW93635_1" class="ff-label"><span class="_ht ht">Enformer</span>:</label><div class="ff-right"><p class="ff-checklist"><input type="checkbox" name="plugin_Enformer" id="CJW93635_1" class="plugin_enable _stt" value="plugin_Enformer"></p></div></div></fieldset>
```